airflow-core/src/airflow/ui/src/layouts/Nav/UserSettingsButton.tsx

```diff
@@ -48,19 +51,38 @@ export const UserSettingsButton = () => {
   return (
     <Menu.Root onOpenChange={() => setTime(dayjs())} positioning={{ placement: "right" }}>
       <Menu.Trigger asChild>
-        <NavButton icon={<FiUser size="1.75rem" />} title="User" />
+        <NavButton icon={<FiUser size="1.75rem" />} title={translate("user")} />
       </Menu.Trigger>
       <Menu.Content>
+        <Menu.Root>
+          <Menu.TriggerItem>
+            <FiGlobe size="1.25rem" style={{ marginRight: "8px" }} />
+            {translate("language.select")}
+            <FiChevronRight size="1.25rem" style={{ marginLeft: "auto" }} />
+          </Menu.TriggerItem>
+          <Menu.Content>
```

Comment:
Instead of using disabled, lets use a [RadioItemGroup](https://www.chakra-ui.com/docs/components/menu#radio-items)